src/logic/dashboard/hooks.ts

```diff
@@ -13,17 +13,22 @@ export const useLoadDashboardData = () => {
   const loadDashboardData = useCallback(async () => {
     if (!provider || !api3Pool || !api3Token || !convenience || !userAccount) return null;
 
-    const [stakingDataErr, stakingData] = await go(convenience.getUserStakingData(userAccount));
-    if (stakingDataErr || !stakingData) {
-      notifications.error({ message: messages.FAILED_TO_LOAD_CHAIN_DATA, errorOrMessage: stakingDataErr! });
+    const goStakingData = await go(convenience.getUserStakingData(userAccount));
```

Comment:
Refactored to `isGoSuccess` pattern for consistency and to make TS more happy (such that I don't need to use `!`)